downstream/modules/platform/proc-configure-known-proxies.adoc

```diff
@@ -10,7 +9,7 @@ To configure a list of known proxies for your {ControllerName}, add the proxy IP
 .Procedure
 //[ddacosta] The Settings > System configurations are for controller only, so don't change ControllerName to PlatformName.
 . From the navigation panel, select {MenuSetSystem}.
-. In the *Proxy IP Allowed List* field, enter IP addresses that are allowed to connect to your {ControllerName}, following the syntax in the example below:
+. In the *Proxy IP Allowed List* field, enter IP addresses that are permitted to connect to your {ControllerName}, following the syntax in the example below:
```

Comment:
For content accessibility, use of directional language should be avoided. Suggest changing to: 
In the *Proxy IP Allowed List* field, enter IP addresses that are permitted to connect to your {ControllerName}, using the syntax in the following example: